diff --git a/third_party/skcms.cmake b/third_party/skcms.cmake
index 61b1ca5..7d967c4 100644
--- a/third_party/skcms.cmake
+++ b/third_party/skcms.cmake
@@ -32,10 +32,10 @@ if(JPEGXL_BUNDLE_SKCMS)
   target_compile_options(skcms-obj PRIVATE -DJPEGXL_BUNDLE_SKCMS=1)
   if(MSVC)
     target_compile_options(skcms-obj
-      PRIVATE /FI${CMAKE_SOURCE_DIR}/lib/jxl/enc_jxl_skcms.h)
+      PRIVATE /FI${CMAKE_CURRENT_SOURCE_DIR}/../lib/jxl/enc_jxl_skcms.h)
   else()
     target_compile_options(skcms-obj
-      PRIVATE -include${CMAKE_SOURCE_DIR}/lib/jxl/enc_jxl_skcms.h)
+      PRIVATE -include${CMAKE_CURRENT_SOURCE_DIR}/../lib/jxl/enc_jxl_skcms.h)
   endif()
 endif()
 
